### ISSUE TITLE
 AYH-6: Ingest JSON feed to populate vector database

### ISSUE THREAD

**Issue by nJim** [yalesites-org/ask-yale-health#6]
Story: As a site owner, I would like the content in our vector database to closely match the content of our website so that the answers of the chat service match our canonical source content. We need a predictable process for populating and updating this database.

## Description of work

- A process is created to update the vector database with content from the JSON feed.
- The process can be automated or scripted if it makes sense at this stage. This can be a content pull on demand or executed on a schedule.
- Out of scope: This will not include work to remove unpublished or deleted content. This will be considered as part of another ticket.

## Acceptance criteria

- [ ] The vector database is populated with content from the JSON feed.
- [ ] A process or set of scripts have been created to update the content.

## Relationships

- Blocked by: https://github.com/yalesites-org/ask-yale-health/issues/3
- Blocked by: https://github.com/yalesites-org/ask-yale-health/issues/4

**Comment by MeganBygnessBradley** [comment]
Test
